python/pyspark/sql/tests/connect/test_connect_plan_only.py

```diff
@@ -58,6 +58,12 @@ def test_join_condition(self):
         )._plan.to_proto(self.connect)
         self.assertIsNotNone(plan.root.join.join_condition)
 
+    def test_crossjoin(self):
+        left_input = self.connect.readTable(table_name=self.tbl_name)
+        right_input = self.connect.readTable(table_name=self.tbl_name)
+        plan = left_input.crossJoin(other=right_input)._plan.to_proto(self.connect)
+        self.assertEqual(plan.root.join.join_type, 7)  # JOIN_TYPE_CROSS
+
```

Comment:
after this PR, `DataFrame.join` also support `cross`, I think we can also add a test for it